### PR TITLE
Spelling overhaul and rev-eng rates for some stuff

### PR DESCRIPTION
**Pull request by Piruzzolo**
Fixed a bunch of typos

### CODE REVIEW

**Comment by zuntrax** [comment]
Nice. You can add yourself to the authors list in `copying.md` if you want to (it's mandatory for source code, but since you only changed documentation, it's your choice).

**Comment by Piruzzolo** [comment]
I mistakenly set my master as the one to pull-request to, so I merged the two in this pull. Sorry for the mess :smile:

**Comment by zuntrax** [comment]
Not much of a problem, but you might save yourself some headaches by pushing your changes to a feature branch like `Piruzzolo:spelling-fix`. Otherwise it easily gets chaotic when opening multiple PRs... :smile:

**Comment by TheJJ** [comment]
I'll take your changes to our upcoming documentation overhaul, then I'll also get rid of your funny merge mess 😛

File: doc/contributing.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ There is no wrongdoing, just wrong restraint.
 You're in luck. There's various sources for tasks:
 
  - have a look at the [issue tracker](https://github.com/sfttech/openage/issues), especially at the issues labeled [easy](https://github.com/SFTtech/openage/labels/easy) or [just do it](https://github.com/SFTtech/openage/labels/just%20do%20it)
- - use [your favourite code searching tool](https://github.com/ggreer/the_silver_searcher) to find `TODO` messages (there's quite a bunch of them)
+ - use [your favorite code searching tool](https://github.com/ggreer/the_silver_searcher) to find `TODO` messages (there's quite a bunch of them)
```
Comment (LordAro) [inline doc/contributing.md]: Nasty nasty US English ;)

Is there actually any decision between one or the other?

Comment (Piruzzolo) [inline doc/contributing.md]: Nope AFAIK. I'm not a native speaker, so feel free to fix weird corrections 👍

**Comment by TheJJ** [comment]
I integrated your corrections and additions in #713, thanks for the patches :)

**Pull request closed without merging**